deployments/local-scripts/relay-messages-millau-to-rialto-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Can you remove the "entrypoint" from the file names? That's a Docker convention that's used, but not relevant to these shell scripts.

Also, a small description of what this script is for would be nice. For example, look at `relay-rialto-to-millau.sh`.